model/src/w3profsmd_pdlib.F90

```diff
@@ -5585,1055 +5529,648 @@ SUBROUTINE PDLIB_JACOBI_GAUSS_SEIDEL_BLOCK(IMOD, FACX, FACY, DTG, VGX, VGY)
+    REAL  :: DMM(0:NK2), DAM(NSPEC), DAM2(NSPEC), SPEC(NSPEC)
+    REAL*8  :: eDiff(NSPEC), eProd(NSPEC), eDiffB(NSPEC)
+    REAL*8  :: DWNI_M2(NK), CWNB_M2(1-NTH:NSPEC)
+    REAL  :: VAnew(NSPEC), VFLWN(1-NTH:NSPEC), JAC, JAC2
     REAL  :: VAAnew(1-NTH:NSPEC+NTH), VAAacloc(1-NTH:NSPEC+NTH)
-    REAL  :: VAinput(NSPEC), VAacloc(NSPEC), eDiffB(NSPEC),ASPAR_DIAG(NSPEC)
-    REAL  :: aspar_diag_local(nspec), aspar_off_diag_local(nspec), b_jac_local(nspec)
-    REAL  :: eDiffSing, eSumPart
-    REAL  :: eVal1, eVal2!, extmp(nspec,nseal)
-    REAL  :: eVA, CG2
-    REAL  :: CG1(0:NK+1), WN1(0:NK+1)
-    LOGICAL :: LCONVERGED(NSEAL), lexist
-#ifdef WEIGHTS
-    INTEGER :: ipiter(nseal), ipitergl(np_global), ipiterout(np_global)
-#endif
-#ifdef W3_DEBUGSRC
-    REAL :: IntDiff, eVA_w3srce, eVAsolve, SumACout
-    REAL :: SumVAin, SumVAout, SumVAw3srce, SumVS, SumVD, VS_w3srce
-    REAL    :: VAsolve(NSPEC)
-    REAL*8  :: ACsolve
-    REAL    :: eB
-#endif
-#ifdef W3_DEBUGSOLVERCOH
-    REAL :: TheARR(NSPEC, npa)
-    REAL :: PRE_VA(NSPEC, npa)
-    REAL :: OffDIAG(NSPEC, npa)
-    REAL*8 :: eOff(NSPEC)
-    REAL*8 :: eSum1(NSPEC), eSum2(NSPEC)
-#endif
-    CHARACTER(len=128) eFile
-    INTEGER ierr, i
-    INTEGER JP_glob
-    INTEGER is_converged, itmp
-
-    integer :: testknoten = 923
-
-    LOGICAL :: LSIG = .FALSE.
-
-    memunit = 50000+IAPROC
-    !AR: this is missing in init ... but there is a design error in ww3_grid with FLCUR and FLLEV
-    LSIG = FLCUR .OR. FLLEV
-#ifdef W3_DEBUGSOLVERCOH
-    OffDIAG = ZERO
-#endif
-    call print_memcheck(memunit, 'memcheck_____:'//' WW3_PROP SECTION 0')
-
-    !DO JSEA = 1, NSEAL
-    !  WRITE(70000+IAPROC,*) 'SUM VA ENTRY SOLVER', JSEA, SUM(VA(:,JSEA))
-    !ENDDO
-
-    CCURX  = FACX
-    CCURY  = FACY
-    CALL MPI_COMM_RANK(MPI_COMM_WCMP, myrank, ierr)
-    !
-#ifdef W3_DEBUGSOLVER
-    WRITE(740+IAPROC,*) 'PDLIB_JACOBI_GAUSS_SEIDEL_BLOCK, begin'
-    WRITE(740+IAPROC,*) 'NX=', NX
-    WRITE(740+IAPROC,*) 'NP=', NP
-    WRITE(740+IAPROC,*) 'NPA=', NPA
-    WRITE(740+IAPROC,*) 'NSEA=', NSEA
-    WRITE(740+IAPROC,*) 'NSEAL=', NSEAL
-    WRITE(740+IAPROC,*) 'NBI=', NBI
-    WRITE(740+IAPROC,*) 'B_JGS_TERMINATE_NORM=', B_JGS_TERMINATE_NORM
-    WRITE(740+IAPROC,*) 'B_JGS_TERMINATE_DIFFERENCE=', B_JGS_TERMINATE_DIFFERENCE
-    WRITE(740+IAPROC,*) 'B_JGS_TERMINATE_MAXITER=', B_JGS_TERMINATE_MAXITER
-    WRITE(740+IAPROC,*) 'B_JGS_MAXITER=', B_JGS_MAXITER
-    WRITE(740+IAPROC,*) 'B_JGS_BLOCK_GAUSS_SEIDEL=', B_JGS_BLOCK_GAUSS_SEIDEL
-    WRITE(740+IAPROC,*) 'FSREFRACTION=', FSREFRACTION
-    WRITE(740+IAPROC,*) 'FSFREQSHIFT=', FSFREQSHIFT
-    WRITE(740+IAPROC,*) 'B_JGS_LIMITER=', B_JGS_LIMITER
-    WRITE(740+IAPROC,*) 'B_JGS_BLOCK_GAUSS_SEIDEL=', B_JGS_BLOCK_GAUSS_SEIDEL
-    FLUSH(740+IAPROC)
-#endif
-#ifdef W3_DEBUGSRC
-    WRITE(740+IAPROC,*) 'optionCall=', optionCall
-    FLUSH(740+IAPROC)
-#endif
-    call print_memcheck(memunit, 'memcheck_____:'//' WW3_PROP SECTION 1')
```

Comment:
@aronroland is there a reason to remove this?

Reply:
@mattewMasarik Aron and I are looking at the code. This is still in the code. Are we looking at the wrong place?

Reply:
@aronroland and @aliabdolali, you're correct, this is now at line 5633.  Cleared.